client/src/components/layouts/DashboardSidebarData.js

```diff
@@ -37,7 +37,12 @@ const DashboardSidebarData = ({ setLogoutAlert }) => {
       name: 'PUBLICATIONS',
     },
     {
-      title: 'Team Member',
+      title: 'Team Profile',
```

Comment:
Can this sidebar item be removed as I think the team profile had been refactored to Account?

Reply:
+1

Reply:
Done